packages/elements/src/components/PoweredByLink.tsx

```diff
@@ -0,0 +1,32 @@
+import { faBolt } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as React from 'react';
+
+export const PoweredByLink: React.FC<{
+  headless?: boolean;
+  source: string;
+  pathname: string;
+  isStoplightProject?: boolean;
```

Comment:
Is there a way to avoid that? I would prefer to avoid mentioning `StoplightProject` in a project that - at least in principle - should know nothing about `StoplightProject`.

Reply:
Could be: package: elements | elements-dev-portal

Reply:
Could also be: `package: string`, then you have 0 coupling.

Reply:
@mmiask yes, I also would prefer what @marcelltoth proposed.